### ISSUE TITLE
[UI] Verify logical errors/ Image Embed no local path to configure

### ISSUE THREAD

**Issue by ZiyiTsang** [bghira/SimpleTuner#1919]
Hi bghira!

I try to use the UI to start the training, however after I configure everthing, the validation fail:
<img width="1146" height="399" alt="Image" src="https://github.com/user-attachments/assets/27974402-a4d2-4d61-824b-6fae2acb5c52" />
I can not find `Model Name` in UI, it is strange...I use AI to search and find out `tracker_project_name` may be reated to this issue. However, I have fill already fill the `tracker_project_name` in json and even `Project Name` in UI...

**So what is `Model Name` stand for?**

The sencond issue is that I find the `local path` loss when config the image embed

<img width="1096" height="741" alt="Image" src="https://github.com/user-attachments/assets/318d4757-030f-48f5-aa92-425e587c28e3" />

Please assist to help,,

**Comment by bghira** [comment]
the storage path for the image embeds is actually specified **on the image or video dataset** - the image embeds is only a storage backend, so that eg. you can point to S3 storage for those, while having image data on Hugging Face Hub.

the storage path must be configured as a part of the image/video dataset, because this allows multiple datasets to share the same cache storage backend without conflicting or all writing into the same dir.

for `Model name is required` I'm not sure why this happened, if you can see it again, please copy the JSON config contents from the button `View JSON` at the top and create an issue with that information.

**Comment by ZiyiTsang** [comment]
[celebrate]     Ziyi Zeng (SDS,224040296) reacted to your message:
________________________________

**Comment by ZiyiTsang** [comment]
[like]  Ziyi Zeng (SDS,224040296) reacted to your message:
________________________________

**Comment by dill-shower** [comment]
> for `Model name is required` I'm not sure why this happened, if you can see it again, please copy the JSON config contents from the button `View JSON` at the top and create an issue with that information.

Hi. I get same error today with latest SimpleTuner. I created new config with wizard and same error on webui and can't start finetune 


`{
  "allow_dataset_oversubscription": false,
  "aspect_bucket_alignment": 64,
  "aspect_bucket_disable_rebuild": false,
  "aspect_bucket_rounding": 2,
  "aspect_bucket_worker_count": 12,
  "attention_mechanism": "diffusers",
  "aws_max_pool_connections": 128,
  "base_model_default_dtype": "bf16",
  "base_model_precision": "no_change",
  "cache_clear_validation_prompts": false,
  "cache_dir_text": "cache",
  "caption_dropout_probability": 0.1,
  "caption_strategy": "filename",
  "checkpoint_step_interval": 100,
  "checkpointing_rolling_steps": 0,
  "checkpointing_steps": 100,
  "checkpointing_use_tempdir": false,
  "checkpoints_rolling_total_limit": 1,
  "checkpoints_total_limit": 5,
  "compress_disk_cache": true,
  "conditioning_multidataset_sampling": "random",
  "context_parallel_comm_strategy": "allgather",
  "context_parallel_size": 1,
  "controlnet": false,
  "createNewDataset": true,
  "data_aesthetic_score": 7,
  "data_backend_config": "zephyr-aurora/multidatabackend.json",
  "data_backend_sampling": "auto-weighting",
  "dataloader_prefetch": false,
  "dataloader_prefetch_qlen": 10,
  "debug_aspect_buckets": false,
  "debug_dataset_loader": false,
  "deepspeed_zero3_init": false,
  "delete_problematic_images": false,
  "delete_unwanted_images": false,
  "disable_benchmark": false,
  "disable_bucket_pruning": false,
  "disable_validations": false,
  "dynamo_backend": "no",
  "dynamo_dynamic": false,
  "dynamo_fullgraph": false,
  "dynamo_mode": "",
  "dynamo_use_regional_compilation": false,
  "ema_cpu_only": false,
  "ema_decay": 0.995,
  "ema_device": "cpu",
  "ema_foreach_disable": false,
  "ema_update_interval": 1,
  "ema_validation": "comparison",
  "enable_group_offload": false,
  "enable_multiprocessing": false,
  "enable_watermark": false,
  "eval_dataset_pooling": false,
  "eval_timesteps": 28,
  "evaluation_type": "none",
  "flux_guidance_max": 4,
  "flux_guidance_min": 0,
  "flux_guidance_mode": "constant",
  "flux_guidance_value": 1,
  "fsdp_activation_checkpointing": false,
  "fsdp_auto_wrap_policy": "TRANSFORMER_BASED_WRAP",
  "fsdp_cpu_offload": false,
  "fsdp_cpu_ram_efficient_loading": true,
  "fsdp_enable": true,
  "fsdp_limit_all_gathers": true,
  "fsdp_reshard_after_forward": true,
  "fsdp_state_dict_type": "SHARDED_STATE_DICT",
  "fsdp_transformer_layer_cls_to_wrap": "",
  "fsdp_version": 2,
  "gradient_accumulation_steps": 2,
  "gradient_checkpointing": true,
  "group_offload_to_disk_path": "",
  "group_offload_use_stream": false,
  "hub_model_id": "",
  "i_know_what_i_am_doing": false,
  "ignore_missing_files": false,
  "image_processing_batch_size": 32,
  "inference_scheduler_timestep_spacing": "trailing",
  "keep_vae_loaded": false,
  "learning_rate": 0.00001,
  "local_rank": -1,
  "logging_dir": "logs",
  "lora_dropout": 0.1,
  "lora_init_type": "default",
  "lora_rank": 16,
  "lora_type": "standard",
  "lycoris_config": "/workspace/simpletuner/configs/config/lycoris_config.json",
  "machine_rank": 0,
  "main_process_ip": "127.0.0.1",
  "main_process_port": 29500,
  "max_train_steps": 99999,
  "max_workers": 32,
  "metadata_update_interval": 3600,
  "mixed_precision": "bf16",
  "model_card_note": "",
  "model_card_private": false,
  "model_card_safe_for_work": false,
  "model_family": "sd3",
  "model_flavour": "3.5-large",
  "model_type": "full",
  "num_eval_images": 4,
  "num_machines": 1,
  "num_processes": 1,
  "num_train_epochs": 0,
  "offload_during_startup": false,
  "only_instance_prompt": false,
  "optimizer": "adamw_bf16",
  "output_dir": "output/zephyr-aurora",
  "override_dataset_config": false,
  "peft_lora_mode": "standard",
  "prepend_instance_prompt": false,
  "preserve_data_backend_cache": false,
  "pretrained_evaluation_model_name_or_path": "openai/clip-vit-large-patch14-336",
  "pretrained_model_name_or_path": "stabilityai/stable-diffusion-3.5-medium",
  "pretrained_transformer_subfolder": "transformer",
  "pretrained_unet_subfolder": "unet",
  "pretrained_vae_model_name_or_path": "madebyollin/sdxl-vae-fp16-fix",
  "print_filenames": false,
  "print_sampler_statistics": false,
  "push_to_hub": false,
  "quantize_activations": false,
  "quantize_via": "accelerator",
  "read_batch_size": 25,
  "refiner_training": false,
  "refiner_training_invert_schedule": false,
  "refiner_training_strength": 0.2,
  "report_to": "wandb",
  "resolution": 1024,
  "resolution_type": "pixel_area",
  "resume_from_checkpoint": "latest",
  "sageattention_usage": "inference",
  "same_network": true,
  "sdxl_refiner_uses_full_range": false,
  "seed": 42,
  "seed_for_each_device": true,
  "singlora_ramp_up_steps": 0,
  "skip_file_discovery": "",
  "text_encoder_1_precision": "no_change",
  "text_encoder_2_precision": "no_change",
  "text_encoder_3_precision": "no_change",
  "text_encoder_4_precision": "no_change",
  "torch_num_threads": 8,
  "tracker_image_layout": "gallery",
  "tracker_project_name": "",
  "tracker_run_name": "",
  "train_batch_size": 4,
  "use_dora": false,
  "use_ema": false,
  "vae_batch_size": 4,
  "vae_cache_ondemand": false,
  "vae_cache_scan_behaviour": "recreate",
  "vae_dtype": "bf16",
  "vae_enable_slicing": false,
  "vae_enable_tiling": false,
  "validation_adapter_mode": "adapter_only",
  "validation_adapter_strength": 1,
  "validation_disable": false,
  "validation_disable_unconditional": false,
  "validation_guidance": 7.5,
  "validation_guidance_skip_layers_start": 0.01,
  "validation_guidance_skip_layers_stop": 0.2,
  "validation_guidance_skip_scale": 2.8,
  "validation_lycoris_strength": 1,
  "validation_multigpu": "batch-parallel",
  "validation_negative_prompt": "blurry, cropped, ugly",
  "validation_num_inference_steps": 20,
  "validation_on_startup": false,
  "validation_prompt": "hatsune miku, 1girl ",
  "validation_prompt_library": false,
  "validation_randomize": false,
  "validation_resolution": "1024x1024",
  "validation_seed_source": "cpu",
  "validation_step_interval": 100,
  "validation_steps": 10,
  "validation_stitch_input_location": "left",
  "validation_torch_compile": false,
  "validation_torch_compile_mode": "max-autotune",
  "validation_using_datasets": false,
  "wan_force_2_1_time_embedding": false,
  "write_batch_size": 128
}`
Connected
 Validation Failed
Model name is required.
 Cannot Start Training
Model name is required.


But another config is work. This is

`{
  "adam_beta1": 0.9,
  "adam_beta2": 0.999,
  "adam_epsilon": 1e-8,
  "adam_weight_decay": 0.01,
  "allow_dataset_oversubscription": false,
  "aspect_bucket_alignment": 64,
  "aspect_bucket_disable_rebuild": false,
  "aspect_bucket_rounding": 2,
  "aspect_bucket_worker_count": 12,
  "attention_mechanism": "diffusers",
  "aws_max_pool_connections": 128,
  "base_model_default_dtype": "bf16",
  "cache_clear_validation_prompts": false,
  "cache_dir_text": "cache",
  "caption_dropout_probability": 0,
  "caption_strategy": "textfile",
  "checkpoint_step_interval": 100,
  "checkpointing_rolling_steps": 0,
  "checkpointing_use_tempdir": false,
  "checkpoints_rolling_total_limit": 1,
  "checkpoints_total_limit": 5,
  "compress_disk_cache": true,
  "conditioning_multidataset_sampling": "random",
  "context_parallel_comm_strategy": "allgather",
  "context_parallel_size": 2,
  "controlnet": false,
  "data_aesthetic_score": 7,
  "data_backend_config": "/workspace/simpletuner/configs/Sd35/multidatabackend.json",
  "data_backend_sampling": "auto-weighting",
  "dataloader_prefetch": true,
  "dataloader_prefetch_qlen": 10,
  "debug_aspect_buckets": false,
  "debug_dataset_loader": false,
  "delete_problematic_images": true,
  "delete_unwanted_images": false,
  "disable_benchmark": false,
  "disable_bucket_pruning": false,
  "disable_segmented_timestep_sampling": false,
  "disable_tf32": false,
  "dynamo_backend": "inductor",
  "dynamo_dynamic": false,
  "dynamo_fullgraph": false,
  "dynamo_mode": "",
  "dynamo_use_regional_compilation": false,
  "ema_cpu_only": true,
  "ema_decay": 0.999,
  "ema_device": "cpu",
  "ema_foreach_disable": false,
  "ema_update_interval": 50,
  "ema_validation": "comparison",
  "enable_group_offload": false,
  "enable_multiprocessing": true,
  "enable_watermark": false,
  "eval_dataset_pooling": false,
  "eval_timesteps": 28,
  "evaluation_type": "none",
  "flow_beta_schedule_alpha": 2,
  "flow_beta_schedule_beta": 2,
  "flow_schedule_auto_shift": false,
  "flow_schedule_shift": 3,
  "flow_sigmoid_scale": 1,
  "flow_use_beta_schedule": false,
  "flow_use_uniform_schedule": true,
  "flux_guidance_max": 4,
  "flux_guidance_min": 0,
  "flux_guidance_mode": "constant",
  "flux_guidance_value": 1,
  "fsdp_auto_wrap_policy": "TRANSFORMER_BASED_WRAP",
  "fsdp_cpu_ram_efficient_loading": false,
  "fsdp_enable": true,
  "fsdp_reshard_after_forward": true,
  "fsdp_state_dict_type": "SHARDED_STATE_DICT",
  "fsdp_transformer_layer_cls_to_wrap": "JointTransformerBlock",
  "fsdp_version": 2,
  "fuse_optimizer": false,
  "grad_clip_method": "norm",
  "gradient_accumulation_steps": 1,
  "gradient_checkpointing": true,
  "group_offload_blocks_per_group": 1,
  "group_offload_to_disk_path": "",
  "group_offload_type": "block_level",
  "group_offload_use_stream": false,
  "huber_c": 0.1,
  "huber_schedule": "snr",
  "i_know_what_i_am_doing": true,
  "ignore_final_epochs": false,
  "ignore_missing_files": false,
  "image_processing_batch_size": 32,
  "inference_scheduler_timestep_spacing": "trailing",
  "input_perturbation": 0,
  "input_perturbation_steps": 0,
  "keep_vae_loaded": false,
  "learning_rate": 0.0001,
  "local_rank": -1,
  "logging_dir": "logs",
  "lora_dropout": 0.1,
  "lora_init_type": "default",
  "lora_rank": 16,
  "lora_type": "standard",
  "loss_type": "l2",
  "lr_end": 4e-7,
  "lr_num_cycles": 10,
  "lr_power": 0.8,
  "lr_scale": false,
  "lr_scale_sqrt": false,
  "lr_scheduler": "cosine_with_restarts",
  "lr_warmup_steps": 160,
  "lycoris_config": "/workspace/simpletuner/configs/config/lycoris_config.json",
  "machine_rank": 0,
  "main_process_ip": "127.0.0.1",
  "main_process_port": 29500,
  "masked_loss_probability": 1,
  "max_grad_norm": 2,
  "max_train_steps": 0,
  "max_workers": 32,
  "maximum_image_size": -1,
  "metadata_update_interval": 3600,
  "mixed_precision": "bf16",
  "model_family": "sd3",
  "model_flavour": "large",
  "model_type": "full",
  "num_eval_images": 4,
  "num_machines": 1,
  "num_processes": 8,
  "num_train_epochs": 1000,
  "offload_during_startup": false,
  "only_instance_prompt": false,
  "optimizer": "prodigy",
  "optimizer_beta1": 0.9,
  "optimizer_beta2": 0.999,
  "optimizer_offload_gradients": false,
  "optimizer_release_gradients": false,
  "output_dir": "/workspace/simpletuner/output",
  "override_dataset_config": false,
  "peft_lora_mode": "standard",
  "prepend_instance_prompt": false,
  "preserve_data_backend_cache": false,
  "pretrained_evaluation_model_name_or_path": "openai/clip-vit-large-patch14-336",
  "pretrained_model_name_or_path": "stabilityai/stable-diffusion-3.5-large",
  "pretrained_transformer_subfolder": "transformer",
  "pretrained_unet_subfolder": "unet",
  "pretrained_vae_model_name_or_path": "/workspace/Vae-checkpoints/vae_clean.safetensors",
  "print_filenames": false,
  "print_sampler_statistics": false,
  "quantize_activations": false,
  "read_batch_size": 50,
  "refiner_training": false,
  "refiner_training_invert_schedule": false,
  "refiner_training_strength": 0.2,
  "report_to": "wandb",
  "resolution": 1024,
  "resolution_type": "pixel_area",
  "resume_from_checkpoint": "latest",
  "sageattention_usage": "inference",
  "same_network": true,
  "sdxl_refiner_uses_full_range": false,
  "seed": 42,
  "seed_for_each_device": true,
  "set_grads_to_none": false,
  "singlora_ramp_up_steps": 0,
  "skip_file_discovery": "",
  "snr_weight": 1,
  "target_downsample_size": 1024,
  "text_encoder_2_precision": "no_change",
  "text_encoder_3_precision": "no_change",
  "timestep_bias_begin": 0,
  "timestep_bias_end": 1000,
  "timestep_bias_multiplier": 1,
  "timestep_bias_portion": 0.25,
  "timestep_bias_strategy": "range",
  "torch_num_threads": 8,
  "tracker_image_layout": "gallery",
  "tracker_project_name": "simpletuner",
  "tracker_run_name": "simpletuner-testing",
  "train_batch_size": 1,
  "training_scheduler_timestep_spacing": "trailing",
  "use_dora": false,
  "use_ema": false,
  "vae_batch_size": 2,
  "vae_cache_ondemand": true,
  "vae_cache_scan_behaviour": "sync",
  "vae_dtype": "bf16",
  "vae_enable_slicing": false,
  "vae_enable_tiling": false,
  "validation_adapter_mode": "adapter_only",
  "validation_adapter_strength": 1,
  "validation_disable": false,
  "validation_disable_unconditional": false,
  "validation_guidance": 4,
  "validation_guidance_skip_layers_start": 0.01,
  "validation_guidance_skip_layers_stop": 0.2,
  "validation_guidance_skip_scale": 2.8,
  "validation_lycoris_strength": 1,
  "validation_negative_prompt": "blurry, cropped, ugly",
  "validation_noise_scheduler": "euler",
  "validation_num_inference_steps": 30,
  "validation_on_startup": false,
  "validation_prompt": "general, 1girl, absurdres, aqua bow, aqua bowtie, black hair, blazer, bow, bowtie, breasts, fang, grey thighhighs, hairband, highres, jacket, kantai collection, large breasts, long hair, multicolored hair, naganami (kancolle), naganami kai ni (kancolle), pink hair, qqqmei, school uniform, smile, solo, thighhighs, twitter username, two-tone hair, wavy hair, white background, white hairband, yellow eyes",
  "validation_prompt_library": false,
  "validation_randomize": false,
  "validation_resolution": "1024",
  "validation_seed": 1642,
  "validation_seed_source": "cpu",
  "validation_step_interval": 10,
  "validation_stitch_input_location": "left",
  "validation_torch_compile": false,
  "validation_torch_compile_mode": "max-autotune",
  "validation_using_datasets": true,
  "wan_force_2_1_time_embedding": false,
  "write_batch_size": 128
}`

**Comment by bghira** [comment]
it's because you have tracker run name and tracker project name blank
